packages/terra-popup/src/Popup.jsx

```diff
@@ -0,0 +1,370 @@
+import React from 'react';
```

Comment:
+1 This is a pretty big file. Separating the logic out would make it more readable